README.md

```diff
@@ -347,16 +359,55 @@ translates to:
 * TooManyStatements#max_statements is 10 in "app/controllers"
 * TooManyStatements#max_statements is 5 everywhere else
 
-For more details please check out the [Basic Smell Options](docs/Basic-Smell-Options.md)
-which are supported by every smell type. As you can see above, certain smell
-types offer a configuration that goes beyond that of the basic smell options, for instance
+Every smell detector supports our [Basic Smell Options](docs/Basic-Smell-Options.md). As you can see above,
+certain smell types offer a configuration that goes beyond that of the basic smell options, for instance
 [Data Clump](docs/Data-Clump.md).
 All options that go beyond the [Basic Smell Options](docs/Basic-Smell-Options.md)
 are documented in the corresponding smell type /docs page (if you want to get a quick overview over all possible
-configurations you can also check out [the `docs/default.reek` file in this repository](docs/defaults.reek).
+configurations you can also check out [the `defaults.reek.yml` file in this repository](defaults.reek.yml).
 
 Note that you do not need a configuration file at all.
-If you're fine with all the [defaults](docs/defaults.reek) we set you can skip this completely.
+If you're fine with all the [defaults](defaults.reek.yml) we set you can skip this completely.
+
+Don't worry about introducing a mistake in your configuration file that might go unnoticed - Reek uses a
+schema to validate your configuration against on start up and will faily loudly in case you
+misspelled an option or used the wrong data type for a value like this:
+
+```
+Error: We found some problems with your configuration file: [/detectors/DetectorWithTypo] key 'DetectorWithTypo:' is undefined.
+```
+
+### Beware of multiple configuration files
```

Comment:
This whole section below was just moved up because I believe this is a nicer flow.

Reply:
Good point, removed the heading